1.17/Makefile.COMMON

```diff
@@ -15,7 +15,7 @@ REPOSITORY    := quay.io/prometheus
 NAME          := golang-builder
 BRANCH        := $(shell git rev-parse --abbrev-ref HEAD)
 SUFFIX        ?= -$(subst /,-,$(BRANCH))
-VERSION       := 1.15.15
+VERSION       := 1.17.0
```

Comment:
Hmm, this should be `1.17`

Reply:
Comparing to #114, I think it's fine. I've pushed a follow-up commit to fix the README though.